mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -113,6 +120,60 @@ properties:
       - 'DSTU2'
       - 'STU3'
       - 'R4'
+  - name: 'consentConfig'
+    type: NestedObject
+    description: |
+      Specifies whether this store has consent enforcement. Not available for DSTU2 FHIR version due to absence of Consent resources. Not supported for R5 FHIR version.
+    min_version: beta
+    properties:
+      - name: 'version'
+        type: Enum
+        description: |
+          Specifies which consent enforcement version is being used for this FHIR store. This field can only be set once by either [fhirStores.create][] or [fhirStores.patch][]. After that, you must call [fhirStores.applyConsents][] to change the version.
+        required: true
+        enum_values:
+          - 'CONSENT_ENFORCEMENT_VERSION_UNSPECIFIED'
+          - 'V1'
+      - name: 'accessEnforced'
+        type: Boolean
+        description: |
+          The default value is false. If set to true, when accessing FHIR resources, the consent headers will be verified against consents given by patients. See the ConsentEnforcementVersion for the supported consent headers.
+      - name: 'consentHeaderHandling'
+        type: NestedObject
+        description: |
+          Different options to configure the behaviour of the server when handling the X-Consent-Scope header.
+        properties:
+          - name: 'profile'
+            type: Enum
+            description: |
+              Specifies the default server behavior when the header is empty. If not specified, the ScopeProfile.PERMIT_EMPTY_SCOPE option is used.
+            enum_values:
+              - 'SCOPE_PROFILE_UNSPECIFIED'
+              - 'PERMIT_EMPTY_SCOPE'
+              - 'REQUIRED_ON_READ'
+            default_value: "PERMIT_EMPTY_SCOPE"
+      - name: 'accessDeterminationLogConfig'
+        type: NestedObject
+        description: |
+          Specifies how the server logs the consent-aware requests. If not specified, the AccessDeterminationLogConfig.LogLevel.MINIMUM option is used.
+        properties:
+          - name: 'logLevel'
+            type: Enum
+            description: |
+              Controls the amount of detail to include as part of the audit logs.
+            enum_values:
+              - 'LOG_LEVEL_UNSPECIFIED'
+              - 'DISABLED'
+              - 'MINIMUM'
+              - 'VERBOSE'
+            default_value: "MINIMUM"
```

Comment:
where does it state that the default_value is MINIMUM?

[docs](https://cloud.google.com/healthcare-api/docs/reference/rest/v1beta1/projects.locations.datasets.fhirStores#loglevel) dont include a default value